packages/nc-gui/assets/style.scss

```diff
@@ -36,6 +36,9 @@ body {
   overflow-y: visible !important;
 }
 
+.rc-virtual-list-holder-inner{
```

Comment:
add a space before `{`

Reply:
**Fixed in below commit**

[fix : removed unwanted spaces](https://github.com/nocodb/nocodb/pull/7181/commits/4b8ee083f5689fd543f5a727c98a384e9f250525)
[4b8ee08](https://github.com/nocodb/nocodb/pull/7181/commits/4b8ee083f5689fd543f5a727c98a384e9f250525)